06_puppy/sirigithub/stores_test.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type storesSuite struct {
+	suite.Suite
+	store     Storer
+	newStorer func() Storer
+	puppy1    Puppy
+}
+
+func (suite *storesSuite) SetupTest() {
+	suite.store = suite.newStorer()
+	suite.puppy1 = Puppy{ID: 1, Breed: "cavalier", Color: "White", Value: "$1700"}
+	er := suite.store.CreatePuppy(&suite.puppy1)
```

Comment:
err

Reply:
Reworked logic of above code Create returns the id of the new puppy and does not throw an error.